lib/utils/compose_ts.ts

```diff
@@ -50,10 +51,10 @@ export async function parseRegistrySecrets(
 		MultiBuild.addCanonicalDockerHubEntry(registrySecrets);
 		return registrySecrets;
 	} catch (error) {
-		error.message =
+		return exitWithExpectedError(
 			`Error validating registry secrets file "${secretsFilename}":\n` +
-			error.message;
-		throw error;
+				error.message,
```

Comment:
Seems a bit strange to use a template literal and then concat?
```typescript
return exitWithExpectedError(
	`Error validating registry secrets file "${secretsFilename}":\n${error.message}`
);
```

Reply:
Reformatted